### ISSUE TITLE
[firefox] Etherpad is randomly unfocused during the call

### ISSUE THREAD

**Issue by jaragunde** [jitsi/jitsi-meet#509]
During a conference with several participants using the Etherpad, the participant using Firefox is sometimes taken out of the pad and the focus is set to the person speaking. This did not happen to the other participants who were using Chrome.

There were no people joining/leaving so it's not the same bug as #48, but may be related.

**Comment by gpolitis** [comment]
Hi @jaragunde, when the issue happens again, could you please take a screenshot and post it here so that we have a visual of the problem? It would also be helpful to have the JavaScript console logs.

**Comment by jaragunde** [comment]
@gpolitis I'll definitely post the JS logs, but the screenshot would be useless, though. What happens is just like pressing the "shared document" button, the pad disappears and one of the video streams get the focus again.

**Comment by wolftune** [comment]
This happens to me on Chromium too, not Firefox specific

**Comment by damencho** [comment]
This was recently fixed (like previous week or the week before that not
sure), which version of meet are you using?

**Comment by wolftune** [comment]
@damencho I only experienced this last week or before, not recently. Maybe it's not happening any more. I didn't have a recent experience

**Comment by jaragunde** [comment]
@damencho Thanks a lot, we'll try it in our next meetings.

**Comment by jaragunde** [comment]
We haven't been able to reproduce this again. Thanks!

**Comment by ryanmartinneutrino** [comment]
I and my work colleagues can reproduce this now (May 2020), with the official docker container. I'm using firefox, but everyone on the call had the issue, and a variety of browsers and OS's. The issue being that the "shared document" loses focus on the big screen if a different person starts speaking than the last one to speak before the document was opened.

I'm running using the docker-compose file form the official docker repository,  and the only variation that I have is that I'm connecting to a separate etherpad container that I've set up independently from the one proposed in the official jitsi docker. We pulled the official docker image around May 18 2020.
